View Controllers/FeedViewController.m

```diff
@@ -113,16 +127,30 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
     
     if(event != nil)
     {
-        if(event.eventType == ETCheckin)
```

Comment:
does a switch work on enums? i forget, but if it does, try a switch statement here instead :)